docs/src/main/asciidoc/resteasy-reactive.adoc

```diff
@@ -1353,6 +1353,15 @@ Importing this module will allow HTTP message bodies to be read from XML
 and serialised to XML, for <<resource-types,all the types not already registered with a more specific
 serialisation>>.
 
+The JAXB resteasy reactive extension will automatically detect the classes that are used in the resources and require of a JAXB serialization. Then, it will register these classes into the default JAXBContext which is internally used by the JAXB message reader and writer. 
```

Comment:
```suggestion
The JAXB Resteasy Reactive extension will automatically detect the classes that are used in the resources and require JAXB serialization. Then, it will register these classes into the default `JAXBContext` which is internally used by the JAXB message reader and writer. 
```